src/request.rs

```diff
@@ -16,11 +19,87 @@ use toml::Table;
 use crate::{
     env::{update_data, HitmanCookieJar},
     extract::extract_variables,
-    prompt::{get_interaction, substitute_interactive},
+    prompt::{get_interaction, prepare_request_interactive},
     util::truncate,
 };
 
-static USER_AGENT: &str = concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_PKG_VERSION"),);
+#[derive(Clone)]
+pub enum HitmanBody {
```

Comment:
I made an enum and implemented two methods on it.

`to_body` to be able to turn it into a body to send as a request. And also implemented `display` to be able to pretty print it in the `print_request` methods for both `hitman-ui` and `hitman`